wandb/sdk/artifacts/artifacts_cache.py

```diff
@@ -85,7 +85,40 @@ def get_client_artifact(self, client_id: str) -> Optional["Artifact"]:
     def store_client_artifact(self, artifact: "Artifact") -> None:
         self._artifacts_by_client_id[artifact._client_id] = artifact
 
-    def cleanup(self, target_size: int, remove_temp: bool = False) -> int:
+    def cleanup(
+        self,
+        target_size: Optional[int] = None,
+        target_fraction: Optional[float] = None,
```

Comment:
This breaks existing callsites, e.g. https://github.com/wandb/wandb/blob/f0caa6367960844aac94c8e9a5e0b329e9422d16/wandb/cli/cli.py#L2208
New arguments can be safely added at the end, but not in the middle.

Reply:
🤦